Formula/chkrootkit.rb

```diff
@@ -1,9 +1,8 @@
 class Chkrootkit < Formula
   desc "Rootkit detector"
   homepage "http://www.chkrootkit.org/"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
```

Comment:
It's worth looking for another mirror if Debian isn't up-to-date. Upstream seems to provide a list here: http://www.chkrootkit.org/mirrors/